### DISCUSSION TITLE
[Feature] Make use of people's names from sidecar

### DISCUSSION THREAD

**Discussion by schnillerman** [immich-app/immich#10585] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Import people's names from sidecar files.

E.g., if photos and people in these photos are managed/tagged in Lightroom, the data is already present in the XMP files, so why should all those available tags be set again in immich.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4348
